packages/system/src/Name/utils.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import {
+  DefaultAutocompleteItemOption,
+  AutocompleteOptionRenderer,
+  Typography,
+} from '@openmsupply-client/common';
+import { NameRowFragment } from './api';
+
+export interface NameSearchProps {
+  open: boolean;
+  onClose: () => void;
+  onChange: (name: NameRowFragment) => void;
+}
+
+export interface NameSearchInputProps {
+  onChange: (name: NameRowFragment) => void;
+  width?: number;
+  value: NameRowFragment | null;
+  disabled?: boolean;
+}
+
+export const simpleNameOptionRenderer: AutocompleteOptionRenderer<
+  NameRowFragment
+> = (props, item) => (
+  <DefaultAutocompleteItemOption {...props}>
+    <Typography sx={{ marginInlineEnd: '10px', fontWeight: 'bold', width: 75 }}>
+      {item.code}
+    </Typography>
+    <Typography>{item.name}</Typography>
+  </DefaultAutocompleteItemOption>
+);
```

Comment:
Is this a good idea? 😁  thought it looked ok..

Reply:
great idea 😁